src/asmcnc/apps/maintenance_app/widget_maintenance_spindle_save.py

```diff
@@ -82,30 +67,30 @@ def __init__(self, **kwargs):
     def get_info(self):
 
         spindle_settings_info = (
+                self.l.get_bold("Spindle brand") + \
+                "[b]: [/b]" + \
+                self.l.get_str("SmartBench will operate slightly differently depending on the type of spindle you are using.") + \
+                " " + \
+                self.l.get_str("It is important that you choose the option that matches the voltage and digital/manual specifications of your spindle.") + \
+                "\n\n" + \
                 self.l.get_bold("Spindle cooldown") + \
                 "[b]: [/b]" + \
                 self.l.get_str("The spindle needs to cool down after a job to prevent it from overheating, and to extend its lifetime.") + \
                 " " + \
                 self.l.get_str("We recommend the following cooldown settings:") + \
-                "\n" + \
+                "\n\n" + \
                 "       " + \
-                self.l.get_str("Yeti: 20,000 RPM; 10 seconds") + \
-                "\n" + \
+                self.l.get_str("Yeti SC1/2: 12,000 RPM; 10 seconds") + \
```

Comment:
non blocking (bc it's an existing bug) but I've just realised that "RPM" and "seconds" should be replaced with their translated variants. Can you address?